packages/deploy/package.json

```diff
@@ -15,7 +15,7 @@
   "homepage": "https://github.com/thesandboxgame/sandbox-smart-contracts#readme",
   "private": true,
   "dependencies": {
-    "@sandbox-smart-contracts/asset": "1.1.0",
+    "@sandbox-smart-contracts/asset": "1.2.0",
```

Comment:
By replacing 1.1.0 by 1.2.0 means we will deploy AssetCreate 1.2.0 and upgrade it to the same version
when we should deploy 1.1.0 then upgrade to 1.2.0.
This is where we have to define how we do when we have 2 versions of the same package

@adjisb can you help on this

Reply:
If I dont change the version its not pulling the newest changes made to AssetCreate in the deploy package =/

Reply:
"@sandbox-smart-contracts/asset-1.1.0": "npm:@sandbox-smart-contracts/asset@1.1.0",
 "@sandbox-smart-contracts/asset-1.2.0": "npm:@sandbox-smart-contracts/asset@1.2.0"

and then:
- you add in hardhat config the files that you want to compile
- use the fully qualified name for your deploy scripts choosing which version you want in each case.

Reply:
So I need to publish AssetCreate@1.2.0 to npm first?